azext_edge/edge/providers/check/base/resource.py

```diff
@@ -468,3 +470,30 @@ def process_custom_resource_status(
                     namespace=namespace,
                     display=Padding(status_text, (0, 0, 0, padding + 4)),
                 )
+
+
+def validate_ref(name: str, namespace: str, ref_type: ValidationResourceType) -> Tuple[str, bool]:
+    ref_obj = None
+    if ref_type == ValidationResourceType.secret:
+        ref_obj = get_namespaced_secret(secret_name=name, namespace=namespace)
+    elif ref_type == ValidationResourceType.configmap:
+        ref_obj = get_config_map(name=name, namespace=namespace)
+
+    return bool(ref_obj)
```

Comment:
If this is in a base module, we may want to raise an exception if `ref_type` is not provided or does not match any of the supported types. My worry is that another module uses this method with a different `ref_type` (like `DataFlowEndpoint`) which always silently returns `False`